runing/routers.py

```
@@ -49,3 +53,41 @@ def delete_user(
 ):
     user_service.delete_user(user_id)
     return {"message":"User deleted successfully"}
+
+@router.post("/items", response_model=schemas.Item)
+def create_item(
+    item: schemas.ItemCreate,
+    item_service: ItemService = Depends(get_item_service)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)